src/axom/inlet/docs/sphinx/advanced_types.rst

```diff
@@ -125,8 +125,8 @@ the following methods:
 
 .. code-block:: C++
 
-  auto coef = inlet["coef"].get<std::function<double(axom::primal::Vector3D)>>();
-  std::function<double(axom::primal::Vector3D)> coef = inlet["coef"];
+  auto coef = inlet["coef"].get<std::function<double(axom::inlet::FunctionType::Vec3D)>>();
```

Comment:
Can these lines be referenced from a compiled and tested example so they don't get out of sync with what works?